src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -395,4 +395,212 @@ public void testIsBooleanLiteralType() {
                 .isFalse();
     }
 
+    @Test
+    public void testAreOnSameLine() {
+        final int type = TokenTypes.COMPILATION_UNIT;
+        final DetailAstImpl astForTest = new DetailAstImpl();
+        final DetailAstImpl astForTest2 = new DetailAstImpl();
+        astForTest.setType(type);
+        astForTest2.setType(type);
+        final boolean result = TokenUtil.areOnSameLine(astForTest, astForTest2);
+
+        assertWithMessage("Token type did not match")
+            .that(result)
+            .isTrue();
+    }
+
+    @Test
+    public void testIsOfTypeTypesTrue() {
+        final int type = TokenTypes.LITERAL_CATCH;
+        final DetailAstImpl astForTest = new DetailAstImpl();
```

Comment:
@Rahulkhinchi03 we try and avoid pure unit tests like this at all costs. Right now, these are just obfuscating dead code, and adding to technical debt. Before we would resort to pure UT, we need an investigation and explanation of why we can't hit a given condition with actual test cases in check tests.

Reply:
All tests should be by inputs, pure Junit for method is absolute last resort with details in comment why it is not possible.

Reply:
@nick-mancuso @romani
I believe that we do not need to create an input file for this. Instead, we can just replace Check Method with the `TokenUtil.java` method.

Reply:
`isOfType()` is overloaded, so EmptyLineSeperatorCheck will use the new method you created for 4 type arguments. No need to rename them.
New inputs are needed just to cover the new methods which are not currently covered. So you should create a test with an input for a check (for example, EmptyLineSeperatorCheck) in order for it to call `isOfType()` method instead of creating tests directly in TokenUtilTest.